test/build_runtime_jar.sh

```diff
@@ -80,8 +84,14 @@ case $BUILD_METHOD in
 	verify)
 	  # mvn clean should clean the target directory, hence using default pom.xml
 	  mvn -f $POM_FILE_NAME clean
+
 	  # mvn verify runs the integration test
-    mvn -f $POM_FILE_NAME verify -Dgpg.skip=true -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
+	  if [BUILD_FOR_CLOUD == "AWS"]; then
+	    # skip Iceberg tests outside of AWS
```

Comment:
i would move the comment to the `else` as here it is misleading a bit

Reply:
Done